jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/RunConfigTestUtils.kt

```diff
@@ -60,7 +59,7 @@ fun executeRunConfiguration(runConfiguration: RunConfiguration, executorId: Stri
                     super.processTerminated(event)
                     // if using the default step executor as the process handle, need to pull text from it or it will be empty
                     // otherwise, pull the output from the process itself
-                    val output = (event.processHandler as? StepExecutor.StepExecutorProcessHandler)?.getFinalOutput() ?: this.output
+                    val output = /* (event.processHandler as? StepExecutor.StepExecutorProcessHandler)?.getFinalOutput() ?: */ this.output
```

Comment:
TODO: Fix this

Reply:
Removed by wrapping the emitters, and forwarding messages onto the process handler